SimMuon/GEMDigitizer/src/GEMSignalModel.cc

```diff
@@ -38,8 +37,6 @@ void GEMSignalModel::simulate(const GEMEtaPartition* roll,
     if (hit.energyLoss() < energyMinCut)
       continue;
     const int bx(getSimHitBx(&hit, engine));
-    if (bx != 0 and bx0filter_)
```

Comment:
Removal of this line will also change the results for Run2 and Run3: please confirm that it is also wanted (possibly also in the PR description, which does not mention it yet)

Reply:
Fortunately, we had the same setting for Run 2 GEM.

Reply:
> Fortunately, we had the same setting for Run 2 GEM.

Does it mean that the changes in Run2/Run3 due to this line removal are wanted, validated and accepted?
Could you please write a few lines in the PR description about it, if so?